app/main.js

```diff
@@ -18,17 +18,40 @@ commander.program
   .option('-a, --all', 'Scrape all airports in all the countries')
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
+const listDisabled = (playbooksDir) => {
+  let disabledPlaybooks = [];
+  try {
+    if (options.listDisabled) {
+      const files = fs.readdirSync(playbooksDir);
+      files.forEach((file) => {
+        const fsPlaybook = JSON.parse(fs.readFileSync(`${playbooksDir}/${file}`, 'utf8'));
+        if (fsPlaybook.enabled === false) {
+          disabledPlaybooks.push(file);
+        }
+      });
+    }
+    logger.info(`https://www.amcharts.com/visited_countries/#${disabledPlaybooks.map((p) => p.split('.')[0]).join(',')}`);
```

Comment:
No need for the chart link. Just the list is enough.

Reply:
So I will put everything inside the block and console.log the list right?

Reply:
Yes.